nio_cli/commands/config.py

```diff
@@ -29,7 +29,8 @@ def config_project(name='.'):
     if (standalone_pubkeeper.lower() == 'n'):
         pk_host = get_pkhost()
         pk_token = get_pktoken()
-        ws_host = pk_host.replace('pubkeeper', 'websocket')
+        ws_host = pk_host.replace('.pubkeeper.', '.websocket.')
+        ws_host = ws_host.replace('pk.demo.', 'ws.demo.')
```

Comment:
What is this for? This doesn't seem like general purpose use to me. If we have customized websocket servers then we should ask the user to enter the websocket server (can default to the replaced pubkeeper server string).

Reply:
I've removed it... I'd rather have some way to denote that a line shouldn't be included in the auto-config process, but I can live with it as is for now.